(EAI-156): `mongodb-chatbot-server` docs + generated reference docs + ingest/optimize rename (#268)
mongodb/chatbot: docs/docs/ingest/optimize.md

### Text

> Using [Markdown](https://www.markdownguide.org/) as the standard format
> is a good choice for most text-based content. LLMs can reason with Markdown,

Reviewer: ```suggestion
[Markdown](https://www.markdownguide.org/) works well as the standard format for most text-based content. LLMs can reason with Markdown,
```

docsy nit to avoid the "using" gerund phrase
